spec/rails_helper.rb

```diff
@@ -147,7 +147,7 @@ def read_csv_from_zip(zip_file, filename)
     end
   end
 
-  config.before do |example|
+  config.before do |_example|
```

Comment:
Why not remove `_example`?

Reply:
Yes, probably a rubocop thing. I removed it